src/library.js

```diff
@@ -993,10 +984,53 @@ LibraryManager.library = {
     }
     return (end-num)|0;
   },
-  llvm_memset_i32: 'memset',
-  llvm_memset_p0i8_i32: 'memset',
-  llvm_memset_p0i8_i64: 'memset',
 
+#if DECLARE_ASM_MODULE_EXPORTS
+  llvm_memcpy_i32: '_memcpy',
```

Comment:
I think these should not have `_`, and it should fail with it, so this seems odd if it passed for you?

Reply:
I did not run this on my system, but instead just let this run through the CI, where it passes. Indeed I agree the code is wrong here, these should be without `_`. The fact that the CI passes suggests that these are not actually used anywhere? Anyhow, fixed to remove `_` now.